pkgs/development/interpreters/dart/default.nix

```
@@ -72,6 +58,7 @@ stdenv.mkDerivation {
   meta = {
     platforms = [ "i686-linux" "x86_64-linux" ];
     homepage = https://www.dartlang.org/;
+    maintainers = with maintainers; [ grburst ];
```

@GRBurst, I think you forgot to use `maintainer` from `stdenv.lib`
```suggestion
    maintainers = with stdenv.lib.maintainers; [ grburst ];
```